Flattening Deeply Nested Data (KSQL) (#191)
confluentinc/kafka-tutorials: _includes/tutorials/flatten-nested-data/ksql/markup/dev/transient-query.adoc

> Now that our continuous query has been created, we can query the result stream called `FLATTENED_ORDERS` with a much simpler query that doesn't need to handle nested data.

Reviewer: ```suggestion
We can query the new result stream called `FLATTENED_ORDERS` with a much simpler query that doesn't need to handle nested data.
```
